### PR TITLE
add Anx

### PR DESCRIPTION
**Pull request by Swingcloud**

### CODE REVIEW

**Comment by tmlee** [comment]
If the API supports market pair (ie. being able to grab a list of pairs via an API), no need to implement the `yml` file. `yml` file would be for exchange that does not support such API, and we have to supply a constant; with the option for Gem user to override it too.

**Comment by Swingcloud** [comment]
sure! will do

**Comment by tmlee** [comment]
@Swingcloud I am sorry, i thought earlier i saw a url in the `pairs` class; i think this is correct!

File: lib/cryptoexchange/exchanges/anx/services/pairs.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          adapt(output)
+        end
+
+        def adapt(output)
+          market_pairs = []
+          output.each do |pair|
+            market_pairs << Anx::Models::MarketPair.new(
+                              base: pair[:base],
+                              target: pair[:target],
+                              market: MARKET::NAME
```
Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/anx/services/pairs.rb]: @Swingcloud change this to `Anx::Market::NAME`

Comment (dominicwong617) [inline lib/cryptoexchange/exchanges/anx/services/pairs.rb]: everything else looks good

File: spec/exchanges/anx/services/pairs_spec.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+require 'spec_helper'
+
+RSpec.describe Cryptoexchange::Exchanges::Anx::Services::Pairs do
```
Comment (dominicwong617) [inline spec/exchanges/anx/services/pairs_spec.rb]: @Swingcloud why is this empty?

**Comment by dominicwong617** [comment]
@tmlee merging this

**Pull request merged**